runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -410,6 +423,30 @@ void contract()
 		pluginPanel = null;
 	}
 
+	void expandSideBar()
+	{
+		if (showSideBar)
```

Comment:
You can't call this 2 times in a row right? so why is the boolean check needed here.